### ISSUE TITLE
Not implemented but exposed options (ignoreTrailingWhiteSpace and ignoreLeadingWhiteSpace)

### ISSUE THREAD

**Issue by HyukjinKwon** [databricks/spark-csv#304]
It looks `ignoreTrailingWhiteSpace` and `ignoreLeadingWhiteSpace` options are exposed but not implemented.

**Comment by HyukjinKwon** [comment]
I think this is related wth https://github.com/databricks/spark-csv/issues/158 if both options mean trimming heading and trailing spaces.

**Comment by HyukjinKwon** [comment]
@falaki Do you intent to implement them or remove them? I will submit a PR if you allow me.

**Comment by falaki** [comment]
Yes, I noticed this problem when I ported the package to Spark 2.0. I think since they are exposed in docs, it is better to implement them.

**Comment by HyukjinKwon** [comment]
Let me try to create a PR first.
(Actually, I just realised they are not exposed in `README.md` but just as parameters in some functions)

**Comment by dereXeus** [comment]
Hi guys, any idea how can I implement this ?
I see that new Parser is being made with settings : 
`new CsvParser(settings)`
and in settings :
`settings.setIgnoreLeadingWhitespaces(ignoreLeadingSpace)`
and default value is :
`ignoreLeadingSpace: Boolean = true`

So when using Univocity , this should be enabled by default ?
